apps/token/src/routes/governance/components/proposals-list/proposals-list.tsx

```diff
@@ -1,84 +1,73 @@
-  proposal: Proposals_proposals;
-}
+  const sortedProposals = failedProposalsCulled.reduce(
+    (acc: SortedProposalsProps, proposal) => {
+      if (isFuture(new Date(proposal.terms.closingDatetime))) {
+        acc.open.push(proposal);
+      } else {
+        acc.closed.push(proposal);
+      }
+      return acc;
+    },
+    {
+      open: [],
+      closed: [],
+    }
+  );
 
-const ProposalListItem = ({ proposal }: ProposalListItemProps) => {
-  const { t } = useTranslation();
-  if (!proposal || !proposal.id) return null;
+  const filterPredicate = (p: Proposals_proposals) =>
+    p.id?.includes(filterString) ||
+    p.party?.id?.toString().includes(filterString) ||
+    p.rationale?.description?.includes(filterString);
 
   return (
-    <li className="last:mb-0 mb-24" key={proposal.id}>
-      <Link to={proposal.id} className="underline text-white">
-        <header>{getProposalName(proposal)}</header>
-      </Link>
-      <KeyValueTable muted={true}>
-        <KeyValueTableRow>
-          {t('state')}
-          <span data-testid="governance-proposal-state">
-            <CurrentProposalState proposal={proposal} />
-          </span>
-        </KeyValueTableRow>
-        <KeyValueTableRow>
-          {isFuture(new Date(proposal.terms.closingDatetime))
-            ? t('closesOn')
-            : t('closedOn')}
-
-          <span data-testid="governance-proposal-closingDate">
-            {format(
-              new Date(proposal.terms.closingDatetime),
-              DATE_FORMAT_DETAILED
-            )}
-          </span>
-        </KeyValueTableRow>
-        <KeyValueTableRow>
-          {isFuture(new Date(proposal.terms.enactmentDatetime))
-            ? t('proposedEnactment')
-            : t('enactedOn')}
-
-          <span data-testid="governance-proposal-enactmentDate">
-            {format(
-              new Date(proposal.terms.enactmentDatetime),
-              DATE_FORMAT_DETAILED
-            )}
-          </span>
-        </KeyValueTableRow>
-      </KeyValueTable>
-    </li>
+    <>
+      <Heading title={t('pageTitleGovernance')} />
+      {failedProposalsCulled.length > 0 && (
+        <ProposalsListFilter setFilterString={setFilterString} />
+      )}
+      {sortedProposals.open.length > 0 ? (
+        <ul data-testid="open-proposals">
+          {sortedProposals.open.filter(filterPredicate).map((proposal) => (
+            <ProposalsListItem key={proposal.id} proposal={proposal} />
+          ))}
+        </ul>
+      ) : (
+        <p data-testid="no-open-proposals">{t('noOpenProposals')}</p>
+      )}
+      <hr className="my-20 border-t-2" />
+      {sortedProposals.closed.length > 0 ? (
+        <ul data-testid="closed-proposals">
+          {sortedProposals.closed.filter(filterPredicate).map((proposal) => (
+            <ProposalsListItem key={proposal.id} proposal={proposal} />
+          ))}
+        </ul>
+      ) : (
+        <p data-testid="no-closed-proposals">{t('noClosedProposals')}</p>
```

Comment:
question (non-blocking): this looks weird at first glance, why are we printing camel case text to the UI? I'd expect some user readable message here, but I don't have much context on this.

Reply:
@notbot00 for context. The token app had a proper translations system set before it got migrated to the monorepo. The `t` function here is from react-i18nnext, whereas the `t` function in other apps (from react-helpers) simply returns the string that was passed in.